src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,649 @@
+
+- ExpressionFilter:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionFilter
+      active: [16]
+      operationMetricsName: AggExpressionFilter
+      pipeline: [{$project: {filter: { $filter: {input: "$array", as: "a", cond: { $eq: ["$$a.b", 0] }}}}}]
+
+- ExpressionFloor:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionFloor
+      active: [17]
+      operationMetricsName: AggExpressionFloor
+      pipeline: [{$project: {floor: { $floor: "$double"}}}]
+
+- ExpressionIfNull:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionIfNull
+      active: [18]
+      operationMetricsName: AggExpressionIfNull
+      pipeline: [{$project: {ifNull: { $ifNull: ["$nullVal", 0]}}}]
+
+- ExpressionIndexOfBytes:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionIndexOfBytes
+      active: [19]
+      operationMetricsName: AggExpressionIndexOfBytes
+      pipeline: [{$project: {indexOfBytes: { $indexOfBytes: ["$hello", "o"]}}}]
+
+- ExpressionIndexOfCP:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionIndexOfCP
+      active: [20]
+      operationMetricsName: AggExpressionIndexOfCP
+      pipeline: [{$project: {indexOfCP: { $indexOfCP: ["$hello", "l"]}}}]
+
+- ExpressionIsNumber:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionIsNumber
+      active: [21]
+      operationMetricsName: AggExpressionIsNumber
+      pipeline: [{$project: {isNumber: { $isNumber: "$randomType"}}}]
+
+- ExpressionLet:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionLet
+      active: [22]
+      operationMetricsName: AggExpressionLet
+      pipeline: [{$project: {let: { $let: {vars: { a: "$int", b: "$int2"}, in: {$gte: ["$$a", "$$b"]}}}}}]
+
+- ExpressionLn:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionLn
+      active: [23]
+      operationMetricsName: AggExpressionLn
+      pipeline: [{$project: {ln: { $ln: "$posInt"}}}]
+
+- ExpressionLog10:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionLog10
+      active: [24]
+      operationMetricsName: AggExpressionLog10
+      pipeline: [{$project: {log10: { $log10: "$posInt"}}}]
+
+- ExpressionMod:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionMod
+      active: [25]
+      operationMetricsName: AggExpressionMod
+      pipeline: [{$project: {mod: { $mod: ["$int", "$posInt"]}}}]
+
+- ExpressionMultiply:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionMultiply
+      active: [26]
+      operationMetricsName: AggExpressionMultiply
+      pipeline: [{$project: {multiply: { $multiply: ["$int", "$int2"]}}}]
+
+- ExpressionNot:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionNot
+      active: [27]
+      operationMetricsName: AggExpressionNot
+      pipeline: [{$project: {not: { $not: [{$gt: ["$int", "$int2"]}]}}}]
```

Comment:
[optional] You could imagine taking the $not of a boolean value in the document directly:

`{$not: ["$myBool"]}`

Reply:
Done.